src/SaveHandler/MongoDBOptions.php

```diff
@@ -13,6 +13,8 @@
 
 /**
  * MongoDB session save handler Options
+ *
+ * @psalm-suppress MissingTemplateParam
```

Comment:
Please remove the inline suppression here and instead add it to the baseline

Reply:
done